core/infrastructure/reconciliation_error.go

```diff
@@ -206,7 +228,6 @@ func (r *reconciliationErrorHandler) GetReconcileResultFor(err error) (ctrl.Resu
 		reconcileError := err.(ReconciliationError)
 		r.Log.Info("Waiting for all resources to be created, re-scheduling.", "reason", reconcileError.reason, "requeueAfter", reconcileError.reconciliationInterval)
 		reconcileResult.RequeueAfter = reconcileError.reconciliationInterval
-		reconcileResult.Requeue = true
```

Comment:
Why is the requeue removed here?

Reply:
It is removed so we can handle no requeue and requeue easily in that code. The [reconciler logic](https://github.com/kubernetes-sigs/controller-runtime/blob/9d7bcf7f784f556882e881c1611240a8a9fdaeb2/pkg/reconcile/reconcile.go#L31) states that if `RequeueAfter` is set, then requeue is automatically true.  We set a`RequeueAfter` interval if and only if it needs to be requeued, so we do not need to explicitly set `Requeue` anymore.

Reply:
ok, thanks